prettier-maven-plugin/src/main/java/com/hubspot/maven/plugins/prettier/PrettierArgs.java

```diff
@@ -108,7 +111,12 @@ protected Path resolveNodeExecutable() throws MojoExecutionException {
       getLog().debug("Resolving node artifact " + nodeArtifact);
     }
 
-    File nodeExecutable = resolve(nodeArtifact).getFile();
+    File nodeExecutable = null;
+    if(nodePath != null && !nodePath.equals("")) {
+      nodeExecutable = new File(nodePath);
```

Comment:
Can you move this to the top of the method? I think we want to early-return when the user supplies a `nodePath` and not do any `setExecutable`. Maybe something like:
```java
if (nodePath != null && !nodePath.isEmpty()) {
  getLog().info("Using nodePath: " + nodePath);
  return Paths.get(nodePath);
}
```